apps/agentic-chat/src/tools/bebopRate.ts

```diff
@@ -72,12 +73,12 @@ export const getBebopRate = async ({
 
   // Convert ETH symbol to Bebop's ETH marker address
   const sellTokenAddress = getAddress(
-    sellAsset.symbol.trim().toUpperCase() === 'ETH'
+    sellAsset.assetId === getFeeAssetByChainId(chainId)
       ? BEBOP_ETH_MARKER
       : fromAssetId(sellAsset.assetId).assetReference
   );
   const buyTokenAddress = getAddress(
-    buyAsset.symbol.trim().toUpperCase() === 'ETH'
+    buyAsset.assetId === getFeeAssetByChainId(chainId)
       ? BEBOP_ETH_MARKER
```

Comment:
I'm assuming this is the same across all EVM chains.

Reply:
Correct!